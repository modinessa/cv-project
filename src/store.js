import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
