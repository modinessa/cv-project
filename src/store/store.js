import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});