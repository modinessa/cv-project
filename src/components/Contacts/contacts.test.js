import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../store/store';
import { Contacts } from './index';

describe('Contacts', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Contacts />
			</Provider>
		)).toMatchSnapshot();
	})
})