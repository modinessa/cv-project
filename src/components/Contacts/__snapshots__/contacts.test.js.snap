// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts renders component 1`] = `
<section
  className="app-section"
>
  <h2>
    Contacts
  </h2>
  <div
    className="contacts-container"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-phone fa-icon-contacts"
      data-icon="phone"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M511.2 387l-23.25 100.8c-3.266 14.25-15.79 24.22-30.46 24.22C205.2 512 0 306.8 0 54.5c0-14.66 9.969-27.2 24.22-30.45l100.8-23.25C139.7-2.602 154.7 5.018 160.8 18.92l46.52 108.5c5.438 12.78 1.77 27.67-8.98 36.45L144.5 207.1c33.98 69.22 90.26 125.5 159.5 159.5l44.08-53.8c8.688-10.78 23.69-14.51 36.47-8.975l108.5 46.51C506.1 357.2 514.6 372.4 511.2 387z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <div
      className="contact-info"
    >
      <a
        href="tel:500-342-0242"
      >
        500 342 242
      </a>
    </div>
  </div>
  <div
    className="contacts-container"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-envelope fa-icon-contacts"
      data-icon="envelope"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M464 64C490.5 64 512 85.49 512 112C512 127.1 504.9 141.3 492.8 150.4L275.2 313.6C263.8 322.1 248.2 322.1 236.8 313.6L19.2 150.4C7.113 141.3 0 127.1 0 112C0 85.49 21.49 64 48 64H464zM217.6 339.2C240.4 356.3 271.6 356.3 294.4 339.2L512 176V384C512 419.3 483.3 448 448 448H64C28.65 448 0 419.3 0 384V176L217.6 339.2z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <div
      className="contact-info"
    >
      <a
        href="mailto:office@kamsolutions.pl"
      >
        office@kamsolutions.pl
      </a>
    </div>
  </div>
  <div
    className="contacts-container"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-twitter fa-icon-contacts"
      data-icon="twitter"
      data-prefix="fab"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M459.4 151.7c.325 4.548 .325 9.097 .325 13.65 0 138.7-105.6 298.6-298.6 298.6-59.45 0-114.7-17.22-161.1-47.11 8.447 .974 16.57 1.299 25.34 1.299 49.06 0 94.21-16.57 130.3-44.83-46.13-.975-84.79-31.19-98.11-72.77 6.498 .974 12.99 1.624 19.82 1.624 9.421 0 18.84-1.3 27.61-3.573-48.08-9.747-84.14-51.98-84.14-102.1v-1.299c13.97 7.797 30.21 12.67 47.43 13.32-28.26-18.84-46.78-51.01-46.78-87.39 0-19.49 5.197-37.36 14.29-52.95 51.65 63.67 129.3 105.3 216.4 109.8-1.624-7.797-2.599-15.92-2.599-24.04 0-57.83 46.78-104.9 104.9-104.9 30.21 0 57.5 12.67 76.67 33.14 23.72-4.548 46.46-13.32 66.6-25.34-7.798 24.37-24.37 44.83-46.13 57.83 21.12-2.273 41.58-8.122 60.43-16.24-14.29 20.79-32.16 39.31-52.63 54.25z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <div
      className="contact-info"
    >
      <h3>
        Twitter
      </h3>
      <a
        className="small-link"
        href="https://twitter.com/wordpress"
      >
        //https://twitter.com/wordpress
      </a>
    </div>
  </div>
  <div
    className="contacts-container"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-facebook fa-icon-contacts"
      data-icon="facebook"
      data-prefix="fab"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.8 90.69 226.4 209.3 245V327.7h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.3 482.4 504 379.8 504 256z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <div
      className="contact-info"
    >
      <h3>
        Facebook
      </h3>
      <a
        className="small-link"
        href="https://www.facebook.com/avdeeva.inessa"
      >
        https://www.facebook.com
      </a>
    </div>
  </div>
  <div
    className="contacts-container"
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-skype fa-icon-contacts"
      data-icon="skype"
      data-prefix="fab"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 448 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M424.7 299.8c2.9-14 4.7-28.9 4.7-43.8 0-113.5-91.9-205.3-205.3-205.3-14.9 0-29.7 1.7-43.8 4.7C161.3 40.7 137.7 32 112 32 50.2 32 0 82.2 0 144c0 25.7 8.7 49.3 23.3 68.2-2.9 14-4.7 28.9-4.7 43.8 0 113.5 91.9 205.3 205.3 205.3 14.9 0 29.7-1.7 43.8-4.7 19 14.6 42.6 23.3 68.2 23.3 61.8 0 112-50.2 112-112 .1-25.6-8.6-49.2-23.2-68.1zm-194.6 91.5c-65.6 0-120.5-29.2-120.5-65 0-16 9-30.6 29.5-30.6 31.2 0 34.1 44.9 88.1 44.9 25.7 0 42.3-11.4 42.3-26.3 0-18.7-16-21.6-42-28-62.5-15.4-117.8-22-117.8-87.2 0-59.2 58.6-81.1 109.1-81.1 55.1 0 110.8 21.9 110.8 55.4 0 16.9-11.4 31.8-30.3 31.8-28.3 0-29.2-33.5-75-33.5-25.7 0-42 7-42 22.5 0 19.8 20.8 21.8 69.1 33 41.4 9.3 90.7 26.8 90.7 77.6 0 59.1-57.1 86.5-112 86.5z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <div
      className="contact-info"
    >
      <h3>
        Skype
      </h3>
      <a
        className="small-link"
        href="skype:echo123?call"
      >
        kamsolutions.pl
      </a>
    </div>
  </div>
</section>
`;
